.github/workflows/release.yaml

```diff
@@ -63,5 +66,6 @@ jobs:
       - 
         name: Update new plugin version in krew-index
         uses: rajatjindal/krew-release-bot@v0.0.40
+        continue-on-error: true
```

Comment:
Why is this option needed here?

Reply:
I discover today that the action return an error when it detects that it is a release candidate
Link to the error: https://github.com/DataDog/datadog-operator/runs/3459789129?check_suite_focus=true